config/app.config

```diff
@@ -23,20 +23,20 @@ debug.mode = false
 
 /*  keep vis.shared.mem.size small if pct is used */
 pct.vis.shared.mem.size = 0.4
-vis.shared.mem.size = "100M"
 ehcache.multicast.ttl = 1
 ehcache.multicast.address = "239.255.0.1"
 
 download.bundle.maxbytes = 304857600
 
 sso.server.url = "https://test.cilogon.org/"
+sso.login.url = "oauth2/start?rd=/portal/suit/"
+sso.logout.url = "/oauth2/sign_in"
```

Comment:
Logout indeed redirects me to `https://lsst-lsp-int.ncsa.illinois.edu/oauth2/sign_in`, but I am not logged out. I can still access `portal/suit` and it shows that I am logged in.